sdk/monitor/Azure.Monitor.Ingestion/tests/ErrorTest.cs

```diff
@@ -12,11 +12,11 @@
 
 namespace Azure.Monitor.Ingestion.Tests
 {
-    [LiveOnly]
+    //[LiveOnly]
```

Comment:
remove this now that it is commented out.

Reply:
this is only for testing the CancellationToken as the PR is still in draft. It will be uncommented to ensure these large tests only run in [LiveOnly] before merging :)